### PR TITLE
VAPI-999: Clarify call update endpoint docs

### PR DESCRIPTION
**Pull request by n-miles**

### CODE REVIEW

**Comment by DX-Bandwidth** [comment]
http://bw-gitbook-api-docs-vapi-999.s3-website-us-east-1.amazonaws.com/

File: voice/methods/calls/postCallsCallId.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Bandwidth's Voice API leverages Basic Authentication with your Dashboard API Cre
...
 |:----------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:----------|
-| state           | (optional) The call state. Possible values: <br><br> `active` to redirect the call (default)<br>`completed` to hangup the call<br>                                                                                                                                  | No        |
+| state           | (optional) The call state. Possible values: <br><br> `active` to redirect the call (default)<br>`completed` to hangup the call or cancel it if it has not been answered<br>                                                                                                                                  | No        |
```
Comment (CatexB) [inline voice/methods/calls/postCallsCallId.md:19]:
```suggestion
| state           | (optional) The call state. Possible values: <br><br> `active` to redirect the call (default)<br>, `completed` to hangup the call, or cancel if it has not been answered<br>                                                                                                                                  | No        |
```

**Review by tmoney-bw — APPROVED** [review]

File: voice/methods/calls/postCallsCallId.md (hunk 1 above)
Comment (n-miles) [reply voice/methods/calls/postCallsCallId.md:19]: `<br>` means line break

Comment (CatexB) [reply voice/methods/calls/postCallsCallId.md:19]: Oh duh. My bad.

**Review by CatexB — APPROVED** [review]

**Pull request merged**